README.md

```diff
@@ -58,7 +58,6 @@ Here's a detailed list of changes/additions that Static Analysis Gradle Plugin m
 
 #### Android Lint
 * Changes the default `abortOnError` flag to false
```

Comment:
add a note that this will copy over the default lintconfig file if there isn't one (and the reason behind it)